2022 - LA - Thousand Sons.cat

```diff
@@ -2653,6 +2665,358 @@
+
+Additionally, a unit that includes any models with this special rule may not be joined by any model that does not also have this special rule (this includes Legion Techmarines and Legion Apothecaries, which may not be assigned to a unit with this special rule unless they also have this special rule)</description>
+            </rule>
+            <rule id="3a90-1892-53a2-3f82" name="Numerologist Order" page="" hidden="false">
+              <description>A model with this special file gains the Psy-synchronicity Psykic Power and may not select a Minor Arcana from those presented by the Prosperine Arcana special rule.</description>
+            </rule>
+          </rules>
+          <infoLinks>
+            <infoLink id="d97e-626e-22d9-c252" name="Relentless" hidden="false" targetId="7adf-ac9a-5035-522d" type="rule"/>
+            <infoLink id="5c52-c6d4-3b40-d7ed" name="Battlesmith (X)" hidden="false" targetId="5d57-4d02-1e36-4a82" type="rule">
+              <modifiers>
+                <modifier type="set" field="name" value="Battlesmith (4+)"/>
+              </modifiers>
+            </infoLink>
+          </infoLinks>
+          <categoryLinks>
+            <categoryLink id="800e-13a8-47de-1127" name="Character" hidden="false" targetId="f75a-d5c1-59ba-5c5a" primary="false"/>
+            <categoryLink id="3f06-e271-542a-0eda" name="Infantry:" hidden="false" targetId="8b4f-bfe2-ce7b-f1b1" primary="false"/>
+          </categoryLinks>
+          <selectionEntryGroups>
+            <selectionEntryGroup id="f524-7b20-49ef-df28" name="1) May take one of the following weapons" hidden="false" collective="false" import="true">
+              <constraints>
+                <constraint field="selections" scope="parent" value="1.0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="5c1e-29c8-4a32-247c" type="max"/>
+              </constraints>
+              <entryLinks>
+                <entryLink id="c982-8a52-79bf-ed69" name="Plasma Gun" hidden="false" collective="false" import="true" targetId="f0d1-332b-c719-ede7" type="selectionEntry">
+                  <costs>
+                    <cost name="Pts" typeId="d2ee-04cb-5f8a-2642" value="15.0"/>
+                  </costs>
+                </entryLink>
+                <entryLink id="4be4-fb04-7db1-9408" name="Bolter" hidden="false" collective="false" import="true" targetId="1ade-0c02-5612-252b" type="selectionEntry">
+                  <modifiers>
+                    <modifier type="set" field="name" value="Master-crafted Bolter"/>
```

Comment:
Set name to master-crafted doesn't work, and the points are set to 10 instead of 5

Reply:
I have since been informed that "set name to master-crafted bolter" does work on the output. I still don't like it, but it can slide. Weapon should still be 5 points though.